prov/efa/src/efa_base_ep.h

```diff
@@ -39,6 +39,26 @@
 #define efa_rx_flags(efa_base_ep) ((efa_base_ep)->util_ep.rx_op_flags)
 #define efa_tx_flags(efa_base_ep) ((efa_base_ep)->util_ep.tx_op_flags)
 
+#define EFA_FABRIC_NAME "efa"
+#define EFA_DIRECT_FABRIC_NAME "efa-direct"
```

Comment:
Can we make these macros defined in efa.h? It's odd to have these info level macros in a base_ep header file

Reply:
I initially put them in `efa.h` but I need them in `efa_domain.h` for `efa_domain_name_suffix`

Will fix alignment